extensions/wikia/Recirculation/js/views/impactFooter.js

```diff
@@ -9,9 +9,7 @@ define('ext.wikia.recirculation.views.impactFooter', [
 
 	var logGroup = 'ext.wikia.recirculation.views.rail',
 		imageRatio = 9/16;
-		options = {
-			template: 'impactFooter.mustache'
-		};
+		options = {};
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "let", "const" or "var" keyword to this declaration of "options" to make it explicit. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=javascript%3AS2703)